spec/dummy/config/application.rb

```diff
@@ -3,7 +3,7 @@
 require 'rails/all'
 
 Bundler.require(*Rails.groups)
-require "rad_core"
+require "mas/rad_core"
```

Comment:
I wanted to pick these all up in another PR. This one just includes the exact changes to accommodate the change in naming.

Also - thanks, Rails generators for your insistence on mixed quotes.